packages/dds/tree/docs/wip/JsonDomainSchema.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Example schema for a json domain.
+ *
+ * Note this is written using the "Example internal schema representation types":
+ * this is not intended to show what authoring a schema would look like, but rather just show what data a schema needs to capture.
+ */
+
+import {
+    TreeSchema,
+    Multiplicity,
+    ValueSchema,
+    FieldSchema,
+    TreeSchemaIdentifier,
+    emptyField,
+    LocalFieldKey,
+    emptyMap,
+    emptySet,
+    NamedTreeSchema,
+} from "./Schema";
+
+export const typeSchema: Map<TreeSchemaIdentifier, TreeSchema> = new Map();
+
+const jsonTypes: Set<TreeSchemaIdentifier> = new Set();
+
+const json: NamedTreeSchema[] = [];
+
+const jsonObject: NamedTreeSchema = {
+    name: "Json.Object" as TreeSchemaIdentifier,
+    localFields: emptyMap,
+    globalFields: emptySet,
+    extraLocalFields: emptyField,
+    extraGlobalFields: false,
+    value: ValueSchema.Nothing,
+};
+
+const jsonArray: NamedTreeSchema = {
+    name: "Json.Array" as TreeSchemaIdentifier,
+    globalFields: emptySet,
+    extraLocalFields: emptyField,
+    extraGlobalFields: false,
+    localFields: new Map([
+        [
+            "children" as LocalFieldKey,
```

Comment:
Nit: Array contents are typically referred to as 'items' rather than 'children'.